mote/templates/aboutpage.html

```diff
@@ -0,0 +1,598 @@
+<!doctype html>
+<!--
+##########################################################################
+*
+*   Copyright © 2019-2021 Akashdeep Dhar <t0xic0der@fedoraproject.org>
+*
+*   This program is free software: you can redistribute it and/or modify
+*   it under the terms of the GNU General Public License as published by
+*   the Free Software Foundation, either version 3 of the License, or
+*   (at your option) any later version.
+*
+*   This program is distributed in the hope that it will be useful,
+*   but WITHOUT ANY WARRANTY; without even the implied warranty of
+*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*   GNU General Public License for more details.
+*
+*   You should have received a copy of the GNU General Public License
+*   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*
+##########################################################################
+-->
+<html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+        <link href="{{ url_for('static', filename='css3/bootstrap.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/custom_fonts.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fragment.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fontawesome-all.min.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/fullcalendar.css') }}" rel="stylesheet"/>
+        <link href="{{ url_for('static', filename='css3/about.css') }}" rel="stylesheet"/>
```

Comment:
Please don't create a new css only for this page. Use the existing one `fragment.css`.